docs/content/en/docs/guidelines-for-contributing.md

```diff
@@ -0,0 +1,35 @@
+## Guidelines for contributing
+
+* Always fork the repository then clone that fork to your local system
+  rather than clone `main` directly.
```

Comment:
After the github contributing page is merged, this should like to the appropriate subsection of that page for directions.